tests/model_hub_tests/models_hub_common/utils.py

```diff
@@ -32,7 +32,8 @@ def compare_two_tensors(ov_res, fw_res, eps):
     is_ok = True
     if not np.allclose(ov_res, fw_res, atol=eps, rtol=eps, equal_nan=True):
         is_ok = False
-        print("Max diff is {}".format(np.array(abs(ov_res - fw_res)).max()))
+        max_diff = np.abs(ov_res.astype(np.float32) - fw_res.astype(np.float32)).max()
```

Comment:
Disagree with this change, let us check that types of both outputs are the same before computation of `max_diff`

Reply:
That is not what this code does. It just reports max diff when np.allclose failed and this will fail for bool outputs, because `-` is not supported for bools